main.js

```diff
@@ -151,6 +202,7 @@
                 record.incidentYear = parseInt(record.INCIDENTTIME.substring(0,4));
                 record.incidentMonth = parseInt(record.INCIDENTTIME.substring(5,8));
                 record.incidentDay = parseInt(record.INCIDENTTIME.substring(8,10));
+                record.type = "police";
```

Comment:
Then, when assigning the .type to record, we could use the dataSourceName parameter in the fetchWPRDCData function